src/actions/authAction.js

```diff
@@ -0,0 +1,75 @@
+import {
+  REGISTER_SUCCESS,
+  REGISTER_FAIL,
+  USER_LOADED,
+  AUTH_ERROR,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  LOGOUT,
+  CLEAR_ERRORS
+} from './types';
+import Requestor from '../components/Requestor';
+import setAuthToken from '../utils/setAuthToken';
+
+const config = {
+  'Content-Type': 'application/json'
+};
+
+export const register = user => async dispatch => {
+  try {
+    const response = await Requestor.post('/users', config, user);
+
+    window.location = '/confirmation-send';
```

Comment:
Rerouting is component specific logic that should not happen in actions. Discoverability of this routing would be deeply hidden otherwise.

Also, from routing, you should pass down `history` variable to `register` component and call `history.push('/confirmation-send')` instead of accessing native javascript `window.location`. Making sure to utilize `history` variable will allow the app to have a rather precinct history state.